cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -134,15 +138,18 @@ class PostgresqlServer(BaseComponent):
     DEGRADED = 1
     DOWN = 2
 
+    upgrading_psql = False
+    encoding = None
+    locale = None
+
     def _init_postgresql_server(self):
         if os.path.exists(PG_HBA_CONF):
             logger.info('PostreSQL Server DATA folder already initialized...')
             return
         logger.debug('Initializing PostgreSQL Server DATA folder...')
-        pg_ctl = join(PGSQL_USR_DIR, 'bin', 'pg_ctl')
-        common.run([
-            'sudo', '-u', 'postgres', pg_ctl, '-D', PGSQL_DATA_DIR, 'initdb'
-        ])
+        common.run(['sudo', '-u', 'postgres',
+                    join(PGSQL_USR_DIR, 'bin', 'initdb'),
+                    '-D', PGSQL_DATA_DIR])
 
         logger.debug('Setting PostgreSQL Server logs path...')
         pg_14_logs_path = join(PGSQL_LIB_DIR, '14', 'data', 'pg_log')
```

Comment:
while you're at it, can you plz also fix this line, in pg14, the dir is called `log`, not `pg_log`